src/content/en/showcase/2016/iowa2016.md

```diff
@@ -0,0 +1,473 @@
+
+<span class="compare-yes"></span> Return users saw a <b>450ms faster first paint</b> thanks to service worker caching
+
+<span class="compare-yes"></span> <b>84%</b> of visitors supported Service Worker
+
+<span class="compare-yes"></span> Add to homescreen saves were up <b>+900%</b> compared to 2015.
+
+<span class="compare-yes"></span> 3.8% users went offline at some point but continued to generate 11k pageviews!
+
+<span class="compare-yes"></span> <b>50%</b> of signed in users enabled notifications.
+
+<span class="compare-yes"></span> <b>536k</b> notifications were sent (12% notifications brought users back to the site).
+
+<span class="compare-yes"></span> <b>99%</b> of users were in a browser that supported the web components polyfills
+
+{% comment %}
+<a hidden class="button button-primary" href="pdfs/FILE.pdf">
+  Download PDF Case study
+</a>
+{% endcomment %}
+
+## Overview
+
+This year, I had the pleasure of [working on](https://events.google.com/io2016/humans.txt) the [Google I/O 2016 progressive web app](https://events.google.com/io2016/), affectionately named "IOWA". It's mobile first, works fully offline, and heavily inspired by [material design](https://www.google.com/design/spec/material-design/introduction.html).
+
+IOWA is a [single page application](https://en.wikipedia.org/wiki/Single-page_application) (SPA) that's built using [web components](http://webcomponents.org/). It pre-caches content
+using a [service worker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/), dynamically loads new pages, gracefully transitions between views, and reuses content after first load.
+
+In this case study, I'll go through some of the more interesting architectural decisions that we made for this year's frontend. If you're interested in the full source code, [check it out](github.com/GoogleChrome/ioweb2016) on Github.
+
+## Building a SPA using web components
+
+IOWA is a [single page application](https://en.wikipedia.org/wiki/Single-page_application) (SPA) that's built using [web components](http://webcomponents.org/). It pre-caches content using a [service worker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/), dynamically loads new pages, gracefully transitions between views, and reuses content after first load.
+
+In this case study, I'll go through some of the more interesting architectural decisions that we made for this year's frontend. If you're interested in the full source code, [check it out](github.com/GoogleChrome/ioweb2016) on Github.
+
+### Every page as a component
+
+One of the core aspects about our frontend is that it's centered around web components. In fact, every page in our SPA is a web component:
+
+    <io-home-page date="2016-05-18T17:00:00Z" app="[[app]]"></io-home-page>
+    <io-schedule-page date="2016-05-18T17:00:00Z" app="{{app}}"></io-schedule-page>
```

Comment:
Django thinks the `{{` in docs is actually meant as a variable, not code. To escape use `{% templatetag openvariable %}{app}}`